### PR TITLE
style(tooltips): FRMW-339 Update Tooltips to Color 2.0

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-339

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

### Before
01_before

### After
02_after

### CODE REVIEW

File: src/tooltips/tooltips.less [hunk 1]
```diff
@@ -7,8 +7,8 @@
 @zindexTooltip: 1070;
```
Comment (ssalinas24) [inline src/tooltips/tooltips.less]: @CITguy Should I move these top few lines into the vars.less file?

**Comment by CITguy** [comment]
FYI: there's actually more contrast between the tooltip bg and the tabs in the "Before"

*before* - more contrast between tooltip gray and tab blue
screen shot 2015-11-18 at 4 39 43 pm

*after* - less contrast between tooltip gray and tab blue. I can barely tell where the tooltip ends and the tab begins.
screen shot 2015-11-18 at 4 40 01 pm

I'm sure this is something that is out of scope of this PR, but maybe adding a white border to the tooltip might be enough to visually separate the two colors.

**Comment by CITguy** [comment]
Code-wise. LGTM

**Comment by glynnis** [comment]
@CITguy FWIW, the tab colors are changing in another PR, so maybe it's worth merging this as-is and checking the contrast of the two once both components are updated. If there's still a contrast problem then, we can open a new story/PR. 

Can you make a note to give the contrast a once-over once tabs and tooltips are merged? I'm working on tabs now but blocked by @ElementE for the moment because we need to add some darker blues to the palette.

@ssalinas24 Not entirely sure just from the screen shots, but it looks like the buttons here are using the old blues. Are you up to date with master? If so, there might be some mark-up on the tooltip demo that needs to be corrected.

**Comment by glynnis** [comment]
Oh. Der. Just realized that the reason the buttons have old colors is because that PR is still open :)

**Comment by CITguy** [comment]
@glynnis, JIRA created (https://jira.rax.io/browse/FRMW-511) to capture request.

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**